website/content/docs/release-notes/consul-ecs/v0_5_x.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: 0.5.x
+description: >-
+  Consul ECS release notes for version 0.5.x
+---
+
+# Consul ECS 0.5.x
+
+## Release Highlights
+
+- **Audit Logging (Enterprise) :** This feature supports audit logging by providing an `audit-logging` flag, providing a log of authenticated events if enabled.
+
+- **Support Consul AWS IAM auth method :** This feature provides support obtaining ACL tokens from Consul's AWS IAM auth method.
+
+- **Mesh Gateways :** This feature introduces Consul WAN federation via mesh gateways ECS tasks, allowing for Consul servers in different datacenters to be federated.
```

Comment:
```suggestion
- **Mesh Gateways :** This feature introduces support for running mesh gateways as ECS tasks. Mesh gateways enable service mesh communication across datacenter and admin partition boundaries.
```

Reply:
I think we can link to the **Configure the gateway task module** topic in the installation page so readers know where to go for usage instructions.

Reply:
Added new phrasing to match @cthain suggestion and linked to appropriate section for usage info in [latest commit](https://github.com/hashicorp/consul/pull/14010/commits/d5f334cf1ee381d5d5b60a4bc33d8f5f99ce9f17)